gulpfile.js

```diff
@@ -1,21 +1,25 @@
+'use strict';
 // Plugins / Functions / Modules
-var gulp    = require('gulp'),
-    plugins = require('gulp-load-plugins')({
-      pattern: ['*', '!gulp', '!gulp-load-plugins'],
-      rename: {
-        'browser-sync'   : 'browserSync',
-        'fs-extra'       : 'fs',
-        'marked-terminal': 'markedTerminal',
-        'run-sequence'   : 'runSequence'
-      }
-    });
+const gulp    = require('gulp'),
+      plugins = require('gulp-load-plugins')({
+        pattern: ['*', '!gulp', '!gulp-load-plugins'],
+        rename : {
+          'browser-sync'   : 'browserSync',
+          'fs-extra'       : 'fs',
+          'marked-terminal': 'markedTerminal',
+          'run-sequence'   : 'runSequence'
+        }
+      });
+
+plugins.errorMessage = require('./helpers/error-message')(plugins);
 
 // Global configuration
-var config = {
+const config = {
   'projectPath': plugins.fs.realpathSync('../../../') + '/'
 };
 
-config.themes = require('./helpers/config-loader')('themes.json', plugins, config);
+config.themes = require('./helpers/config-loader')('themes.json', plugins, config, false);
+plugins.getThemes = require('./helpers/get-themes')(plugins, config);
```

Comment:
The reason I added it this way is to expose the helper with less parameters. Also plugins is already shared across everything.